server/db/db.js

```diff
@@ -521,6 +521,28 @@ module.exports.saveButtonBatteryLevel = async function(serialNumber, batteryLeve
     }
 }
 
+module.exports.updateSessionBatteryLevel = async function(sessionid, batteryLevel, client){
```

Comment:
I have the same feedback here as I did for @trisapeace's BraveAlert integration PR - I don't think we need an extra db function for this, as it is already covered by `db.saveSession()`.